src/UploadedFile.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+namespace React\Http;
+
+use InvalidArgumentException;
+use Psr\Http\Message\StreamInterface;
+use Psr\Http\Message\UploadedFileInterface;
+use RuntimeException;
+
+final class UploadedFile implements UploadedFileInterface
```

Comment:
I don't see this class being used or documented anywhere, should this be marked `@internal`?